app/views/unattended/provisioning_templates/snippet/disk_enc_clevis_tang.erb

```diff
@@ -27,52 +27,66 @@ description: |
 <% if (@host.operatingsystem.family == 'Redhat' || @host.operatingsystem.name == 'Ubuntu') && tang_server_list.present? -%>
 
 cat > /tmp/rootdir-luks-device.sh << "EOF"
-#!/bin/sh
+#!/bin/bash
```

Comment:
why to switch to bash instead of sh?

Reply:
It was already called with `luksdev=$(bash /tmp/rootdir-luks-device.sh)` before, thought to best adapt the shebang to reflect that. The reason is in a already existing comment right where that script is called:

    # needs bash here because Ubuntu's sh (dash) doesn't support `-o pipefail` option
    
 I think `mapfile` calls that I added are another feature that is bash-specific